src/app/core/services/api/api.service.ts

```diff
@@ -80,7 +81,10 @@ export class ApiService {
       // testing token gets 'null' from captcha service, so we accept it as a valid value here
       captchaAction !== undefined
         ? // captcha V3
-          headers.set(ApiService.AUTHORIZATION_HEADER_KEY, `CAPTCHA recaptcha_token=${captcha} action=${captchaAction}`)
+          headers.set(
+            ApiService.RECAPTCHA_TOKEN_HEADER_KEY,
```

Comment:
does this changed token header key require a certain ICM version?

Reply:
The X-Captcha-Token header was introduced with [81572](https://dev.azure.com/intershop-com/Products/_workitems/edit/81572). Fixed versions are 7.10.40.0 and 11.0.12.